hypothesis-python/src/hypothesis/strategies/_internal/numbers.py

```diff
@@ -217,9 +220,10 @@ def permitted(self, f):
         if self.width < 64:
             try:
                 float_of(f, self.width)
-                return True
             except OverflowError:
                 return False
+        if not self.allow_subnormal and 0 < abs(f) < width_smallest_normals[self.width]:
```

Comment:
```suggestion
        if (not self.allow_subnormal) and 0 < abs(f) < width_smallest_normals[self.width]:
```

No semantic difference, but it's a bit easier to read.